kythe/proto/serving.proto

```diff
@@ -30,6 +30,8 @@ message Node {
 
   // This node's definition location (anchor).
   ExpandedAnchor definition_location = 3;
+
+  double rank = 4;
```

Comment:
Can this (in a future change) get some more documentation? E.g., what are the units here? Is there an expected range? Is it a magnitude, a percentage, other? Is higher better? What do fractional values mean?

(I realize these things may still be up for discussion, but I haven't seen any of it so I'm operating blind here)